float8_experimental/float8_tensor.py

```diff
@@ -31,6 +28,20 @@
 )
 
 
+class ScalingStrategy(Enum):
```

Comment:
thoughts about using `Granularity`, which is more specific than `Strategy`?

Reply:
Yeah thats a better word, this needed some bikeshedding